lib/pm.js

```diff
@@ -71,7 +71,15 @@ Pm.prototype.spawn = function(callback) {
 
   mkdirp(this.base);
 
-  var logFd = fs.openSync(this.logFile, 'a');
+  try {
+    fs.truncateSync(this.logFile, 0);
+  } catch(er) {
+    // Ignore, probably the file doesn't exist, and other errors
+    // will be caught below when we open the file.
+    debug('truncate failed: %s', er.message);
+  }
+
+  var logFd = fs.openSync(this.logFile, 'w');
```

Comment:
this `'w'` was a mistake, a relic of an earlier experiment, backed out (in answer to your question above)